### DISCUSSION TITLE
vscode create route folders

### DISCUSSION THREAD

**Discussion by akomm** [remix-run/remix#8902] [Q&A] [answered]
I have >100 folders by now and generally like it, though it becomes painful to create new folders in vscode, because I need to scroll up to the route folder always to be able to create a new folder in it. I could not find any shortcut to handle this or any plugin that helps with that. How do you handle that? Thanks.

**Comment by kiliman** [comment, marked as answer]
Probably the simplest one is `File Utils`. It has a command to create a new folder relative to project root.

https://marketplace.visualstudio.com/items?itemName=sleistner.vscode-fileutils

<img width="603" alt="image" src="https://github.com/remix-run/remix/assets/47168/feddaa0e-6e93-49af-b0d8-9db1bb4b0f69">

<img width="605" alt="image" src="https://github.com/remix-run/remix/assets/47168/5b714965-d275-4121-b59a-2dc21f149e46">

**Reply by akomm** [reply]
Thanks!

I've just tested this. It works quite well. I was not sure first because I thought I have to type app/routes all the time, but it gives you combobox input to quickly find a subfolder within root and then create and it remembers last selection too... smart.
